chain-connect/docs/Examples.md

```diff
@@ -0,0 +1,319 @@
+- [Server-Side Signing](#server-side-signing)
+  - [Server-Side Example](#server-side-example)
+- [Notes](#notes)
+
+## Prerequisites
+
+- Ensure you have installed GalaChain Connect and its dependencies.
+- You have a supported Web3 wallet installed and set up in your browser.
+- Your application is set up to use a frontend framework like Vue.js or React.
+- For server-side examples, ensure you have Node.js installed.
+
+## Client-Side Usage
+
+### Connecting to a Web3 Wallet
+
+First, you need to create a client and connect to a web3 wallet, the below shows a connection:
+
+```typescript
+import { BrowserConnectClient } from "@gala-chain/connect";
+import { ref } from "vue";
+
+const web3Wallet = new BrowserConnectClient();
+export const message = ref("");
+export const connectedUser = ref<string | null>(null);
+
+export async function connectToWeb3Wallet() {
+    try {
+        const connectionResult = await web3Wallet.connect();
+        message.value = `Connected! User: ${connectionResult}`;
+        connectedUser.value = connectionResult;
+
+        // Listening to account changes
+        web3Wallet.on("accountChanged", (account: string[] | string | null) => {
+            if (Array.isArray(account)) {
+                //For simplicity of the example,we'll just grab the first connected wallet
+                if (account.length > 0) {
+                    connectedUser.value = account[0];
+                }
+            } else {
+                console.log(`Account changed: ${account}`);
+                message.value = `Account Changed! User: ${account}`;
+                connectedUser.value = account;
+            }
+        });
+    } catch (error) {
+        message.value = "Failed to connect!";
+        console.error(error);
+    }
+}
+
+export function isConnected() {
+    if (Array.isArray(connectedUser.value)) {
+        return connectedUser.value.length > 0;
+    }
+    return connectedUser.value !== null;
+}
+```
+
+### Creating a Token Class
+
+Once connected to a web3 wallet, you can create a token class using the `TokenApi`:
+
+```typescript
+import { TokenApi } from "@gala-chain/connect";
+
+const tokenClient = new TokenApi("https://your-galachain-api-url/asset/token-contract", web3Wallet);
+
+const arbitraryTokenData = {
+  tokenClass: {
+    additionalKey: "fakeTokenKey",
+    category: "fakeCategory",
+    collection: "fakeCollection",
+    type: "fakeType"
+  },
+  description: "This is a description",
+  image: "foo.png",
+  name: "Foo",
+  symbol: "FOO",
+  maxSupply: "100000"
+};
+
+export async function createTokenClass() {
+  try {
+    const result = await tokenClient.CreateTokenClass(arbitraryTokenData);
+    console.log("Token class created:", result);
+  } catch (error) {
+    console.error("Failed to create token class:", error);
+  }
+}
+```
+
+### Template
+
+In your Vue component, you can add a simple template with a connect button and a button to create a token class:
+
+```html
+<template>
```

Comment:
This is something we could add in the UI package, but UI related examples are unnecessary here

Reply:
Removed!